packages/actor-init-query/test/HttpServiceSparqlEndpoint-test.ts

```diff
@@ -1724,7 +1724,7 @@ describe('HttpServiceSparqlEndpoint', () => {
         httpRequestMock.headers = { 'content-type': 'application/x-www-form-urlencoded' };
 
         return expect(instance.parseBody(httpRequestMock)).rejects
-          .toThrowError(`Invalid POST body with context received ('{"a:"b"}'): Unexpected token b in JSON at position 5`);
+          .toThrowError(/(Invalid POST body with context received \('\{"a:"b"\}'\):) .+JSON (at position 5)/u);
```

Comment:
```suggestion
          .toThrowError('Invalid POST body with context received');
```

I applaud your effort in finding this regex, but I think we can just simplify it like the above, no need to check for the literal JSON error message :-)